File: backend/src/main/java/com/realdolmen/backend/Domain/Game.java
package com.realdolmen.backend.Domain;

import com.realdolmen.backend.Domain.enums.GameGenre;
import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Setter
@Getter
@Entity
@DiscriminatorValue("game")
public class Game extends Article {
    @Column(name = "game_genre")
    @Enumerated(EnumType.STRING)
    private GameGenre gameGenre;

    private Integer minimumAge;

    private String publisher;

}

File: frontend/angular/src/app/domain/game.ts
import {Article} from "./article";

export class Game extends Article{
  constructor(public title: string,
              public type: string,
              public price: number,
              public supplierId: string,
              public gameGenre:string,
              public minimumAge:number,
              public publisher:string,
              public articleId?: string,) {
    super(title, type, price, supplierId, articleId)
  }
}
